pillar.example.sls

```diff
@@ -1,68 +1,93 @@
 postgresql:
   lookup:
+    repo:
```

Comment:
In this case i'm not a fan of having an external deb on repo mgmt on reason is everything is configurable trough pillar another is saltstack-repos-formula requires salt-modules. So one who just wants this formula needs the other two to.